### PR TITLE
[CMS-909] Add WordPress Composer Managed EA docs

### PR DESCRIPTION
**Pull request by jspellman814**
## Summary

Adds EA docs for the WordPress Composer Managed upstream.

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7630](https://pr-7630--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `47` | `45` | `-2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7630` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/41065/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/41065/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7630--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by jspellman814** [comment]
[Preview](https://pr-7630--pantheon-docs.my.pantheonfrontend.website/docs/guides/wordpress-composer/wordpress-composer-managed)

File: source/content/guides/wordpress-composer/01-introduction.md
Comment (CdrMarks) [inline source/content/guides/wordpress-composer/01-introduction.md:36]: Is this still the suggested path for customers?

File: source/content/guides/wordpress-composer/02-wordpress-composer-managed.md
Comment (CdrMarks) [inline source/content/guides/wordpress-composer/02-wordpress-composer-managed.md:16]: Can we link to a definition of Early Access?

File: source/content/guides/wordpress-composer/02-wordpress-composer-managed.md
Comment (CdrMarks) [inline source/content/guides/wordpress-composer/02-wordpress-composer-managed.md:26]: Are there still issues running WP 6.x and PHP 8.1?

Comment (jazzsequence) [reply source/content/guides/wordpress-composer/02-wordpress-composer-managed.md:26]: WP 6.x support for PHP 8.x is in "beta". I'm not sure that's relevant to call out here, I think it overcomplicates things.

**Comment by jms-pantheon** [comment]
Should we remove or clarify the difference between this new guide and the previous one here? 
https://pr-7630--pantheon-docs.my.pantheonfrontend.website/docs/guides/wordpress-composer/wordpress-ic

I would also suggest adding some context around Integrated Composer for those who are new to it but looking to use Bedrock with Pantheon. We could link to the existing Integrated Composer docs after a brief description of how Pantheon will run `composer install` and composer-installed folders should not be committed to the repository.

File: source/content/guides/wordpress-composer/02-wordpress-composer-managed.md
Comment (jazzsequence) [inline source/content/guides/wordpress-composer/02-wordpress-composer-managed.md:50]: typo
```suggestion
Bedrock makes use of an `.env` file to store environment variables. Pantheon takes care of many of these variables in `.env.pantheon`. You may set your own environment variables in a new `.env` or environment variables that are local-only in `.env.local` using the `.env.example` as a guide. Wrap values that may contain non-alphanumeric characters with quotes, or they may be incorrectly parsed.
```

**Review by jazzsequence — APPROVED** [review]
LGTM

**Comment by jazzsequence** [comment]
@jms-pantheon While the Bedrock upstream is still in EA/pre-GA, we just want to have docs that talk about how to use it. Ultimately, all the WordPress Composer-related docs will need updating for the new Bedrock-based upstream and process. I think it's early to update those things now.

File: source/content/guides/wordpress-composer/02-wordpress-composer-managed.md
Comment (jspellman814) [reply source/content/guides/wordpress-composer/02-wordpress-composer-managed.md:16]: Not sure that we can link text in an Alert, but I'll test it out.

Comment (jspellman814) [reply source/content/guides/wordpress-composer/02-wordpress-composer-managed.md:16]: @CdrMarks is this the page you had in mind to link to?

File: source/content/guides/wordpress-composer/02-wordpress-composer-managed.md
Comment (jspellman814) [reply source/content/guides/wordpress-composer/02-wordpress-composer-managed.md:26]: Here's WordPress' PHP version support chart: https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/#supported-version-chart.

Don't think we need to call it out here, as it's mentioned on our own [Supported PHP Version page](https://pantheon.io/docs/supported-php), but I'd make that change if someone felt otherwise.

File: source/content/guides/wordpress-composer/02-wordpress-composer-managed.md
Comment (CdrMarks) [reply source/content/guides/wordpress-composer/02-wordpress-composer-managed.md:16]: @jspellman814 I was thinking https://pantheon.io/docs/oss-support-levels#early-access since this is where the repository badges link.

**Review by CdrMarks — CHANGES_REQUESTED** [review]
One change left for me:
https://github.com/pantheon-systems/documentation/pull/7630#discussion_r955381821

**Review by CdrMarks — APPROVED** [review]

**Review by scottbuscemi — CHANGES_REQUESTED** [review]

File: source/content/guides/wordpress-composer/02-wordpress-composer-managed.md
Comment (whitneymeredith) [reply source/content/guides/wordpress-composer/02-wordpress-composer-managed.md]: @scottbuscemi All numbered steps use 1 and (with Git / Markdown magic) publish with correctly numbered lists.

File: source/content/guides/wordpress-composer/02-wordpress-composer-managed.md
Comment (whitneymeredith) [reply source/content/guides/wordpress-composer/02-wordpress-composer-managed.md]: @scottbuscemi I added the link - thanks!!

**Comment by jspellman814** [comment]
@whitneymeredith If we are going to include the additions from [26afab6](https://github.com/pantheon-systems/documentation/pull/7630/commits/26afab66a840caf65c2ca2116d9fde8f5ce159d7), we will need additional time to review from the technical side. Some of the information included is not entirely accurate as it relates to the new upstream.

**Comment by whitneymeredith** [comment]
@jspellman814 Technical additions from [26afab6](https://github.com/pantheon-systems/documentation/pull/7630/commits/26afab66a840caf65c2ca2116d9fde8f5ce159d7) were removed. The changes you see are the break out of pre-existing content into multiple pages to improve usability / accessibility.

**Comment by jspellman814** [comment]
@whitneymeredith My sincerest apologies - our intention here was to get a smaller guide out for EA for the WordPress Composer Managed upstream. If we are going to include the additions from [26afab6](https://github.com/pantheon-systems/documentation/pull/7630/commits/26afab66a840caf65c2ca2116d9fde8f5ce159d7), we will need additional time to review from the technical side. Some of the information included is not entirely accurate as it relates to the new upstream.

**Comment by jspellman814** [comment]
> @jspellman814 Technical additions from [26afab6](https://github.com/pantheon-systems/documentation/pull/7630/commits/26afab66a840caf65c2ca2116d9fde8f5ce159d7) were removed. The changes you see are the break out of pre-existing content into multiple pages to improve usability / accessibility.

Thanks so much, this is looking great.

General question (maybe requires @scottbuscemi input)... is there some way to group/organize these pages that would make it easier to digest? In the screenshot below, the non-highlighted pages related to WordPress Composer Managed EA, while the others are not applicable. I could see this being confusing to customers.

Screen Shot 2022-08-25 at 3 17 03 PM

**Comment by whitneymeredith** [comment]
@jspellman814 I can re-order the pages. What order do you want them in?

**Comment by CdrMarks** [comment]
@whitneymeredith could you please revert the merging in of the content from #7577?  We'd like to move forward with just what John proposed so we can have documentation in place for EA.  I think it'd be fine to consider merging these pages together, but would like to tackle that in a separate PR.

**Comment by whitneymeredith** [comment]
@CdrMarks It's the same content as before. It's just broken out into multiple pages.

**Comment by CdrMarks** [comment]
@whitneymeredith I am seeing several commits from 16 days ago (https://github.com/pantheon-systems/documentation/pull/7630/commits/07e39cf26fac53ed0af3f0fd23ee0416364402ce) through 7 days ago (https://github.com/pantheon-systems/documentation/pull/7630/commits/b961a2f615a716aa8f3e618d4e13863c8b08a924) that we do not want on this small guide for EA.

Could you please revert the changes you have made.

**Comment by whitneymeredith** [comment]
@CdrMarks It's reverted.

**Comment by whitneymeredith** [comment]
LGTM

**Comment by whitneymeredith** [comment]
@jspellman814 @scottbuscemi  Do you want me to merge this after CI completes testing?

**Comment by scottbuscemi** [comment]
@whitneymeredith yes please! :)

**Pull request merged**
